SmartDeviceLink/SDLCarWindowViewController.h

```diff
@@ -0,0 +1,15 @@
+//
+//  SDLCarWindowViewController.h
+//  SmartDeviceLink
+//
+//  Created by Joel Fischer on 12/12/17.
+//  Copyright © 2017 smartdevicelink. All rights reserved.
+//
+
```

Comment:
I think we should note in the documentation that orientation locking will not work if the `viewController` is embedded in a `UINavigationController` or a `UITabBarController`.

Reply:
Done